public/assets/js/scripts.js

```diff
@@ -294,6 +294,17 @@ $(document).ready(function () {
     $(this).html() == '<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>' ? $(this).html('<span><img src="./assets/img/icons/view-map-pin.svg" alt="view map"> &nbsp; View Map</span>') : $(this).html('<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>');
   });
 
+
+  // disable calculate button 
```

Comment:
I think this comment is misleading. The below code deals more with enabling the calculate button than disabling it. Can we remove it / add an appropriate comment?